### ISSUE TITLE
Support edit access

### ISSUE THREAD

**Issue by adhikasp** [sooperset/mcp-atlassian#12]
Thanks for the project! 

It would be useful to also have create/edit capabality to both JIRA and Confluence page

**Comment by sooperset** [comment]
Definitely! I understand the need for these features. However, due to my current schedule, I'm not sure when I'll be able to implement them. Contributions are always welcome!
